### DISCUSSION TITLE
Possibility of working with values greater than MAX_SAFE_INTEGER

### DISCUSSION THREAD

**Discussion by DOUGLASMD1** [dm4t2/vue-currency-input#387] [Q&A] [answered]
Is there a possibility to work with the entry of coins above MAX_SAFE_INTEGER?

For both decimal and integer values?

For example: 9.999.999.999.999.999.999.999,99

I have a feature in my project that should support up to 24 digits

**Comment by dm4t2** [comment, marked as answer]
Currently not, see #346
